src/processor/map/map_recursive_extend.cpp

```diff
@@ -0,0 +1,64 @@
+#include "graph/on_disk_graph.h"
+#include "planner/operator/extend/logical_recursive_extend.h"
+#include "planner/operator/sip/logical_semi_masker.h"
+#include "processor/operator/gds_call.h"
+#include "processor/plan_mapper.h"
+
+using namespace kuzu::planner;
+using namespace kuzu::graph;
+using namespace kuzu::binder;
+using namespace kuzu::common;
+
+namespace kuzu {
+namespace processor {
+
+std::unique_ptr<PhysicalOperator> PlanMapper::mapRecursiveExtend(
```

Comment:
Will this change soon? If not, I would refactor this function altogether with `mapGDSCall`.